.gitignore

```diff
@@ -32,3 +32,6 @@
 
 # Ignore application configuration
 /config/application.yml
+
+# Ignore configuration of Rubocop(s)
+.rubocop.yml
```

Comment:
Why do you want to ignore rubocop configuration?

Storing rubocop configuration for the whole project is important cause we want to have consistent behavior in all environments for all developers who work on the project. 
Consider that I've cloned your project and ran rubocop in this branch and I see that there we have:
1. 9 warnings
2. 1 error (it's a syntax error!)
3. 1 correctable warning (so it might be automaticall solved)
![image](https://user-images.githubusercontent.com/4128525/129086526-e17d2022-45e4-47e1-89c3-ae3997fd977d.png)

I see this status cause probably you did not upload your updated configuration of rubocop. It's a good idea to not add to git our local text editor settings, but it's an awesome idea to always add to git configuration of tools (or gems) that we using in the project.

Also, consider that you can ignore in rubocop configuration all files from directory `bin`, you can easily remove at all `test` directory and then you will have only 2 warnings, where both might be easily solved -> [article](https://thedevpost.com/blog/fix-rubocop-missing-comment/)

Reply:
I have to admit - I didn't think this through - or rather I've had the wrong idea about rubocop, treating it as my local tool.

As you've already guessed - everything in here is handled by my local rubocop configuration file.
I also do not care right now about the `test` directory, because it is redundant and it will be replaced by rspec anyway, but I figured it's not the place to deal with it (since it's a branch for Sentry configuration - although I've been fixing unrelated rubocop offenses here as well...).

I'm also aware of one more way to handle ignoring some parts of the code - by doing so directly in the source code with, for example, [disable/enable rubocop comments](https://docs.rubocop.org/rubocop/configuration.html#disabling-cops-within-source-code) disabling cops, departments, or the entire rubocop, but since I've looked at rubocop in the wrong way it seemed superfluous back then. Now I know better.

It is now all fixed.

Thank you for the feedback!